crates/apps/src/lib/cli.rs

```diff
@@ -3034,8 +3034,6 @@ pub mod args {
             Err(_) => config::get_default_namada_folder(),
         }),
     );
-    pub const BATCH_SIZE_OPT: ArgDefault<u64> =
```

Comment:
we will still want this for shielded sync, no?

Reply:
Yes my bad, removed this by mistake